src/amo/components/StaticPages/ReviewGuide.js

```diff
@@ -0,0 +1,117 @@
+import Helmet from 'react-helmet';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { compose } from 'redux';
+
+import Card from 'ui/components/Card';
+import translate from 'core/i18n/translate';
+import { sanitizeHTML } from 'core/utils';
+
+import 'amo/components/StaticPages/StaticPages.scss';
+
+
+export class ReviewGuideBase extends React.Component {
+  static propTypes = {
+    i18n: PropTypes.object.isRequired,
+  }
+
+  render() {
+    const { i18n } = this.props;
+
+    return (
+
+      <Card
+        className="StaticPage"
+        header={i18n.gettext('Review Guidelines')}
+      >
+        <Helmet>
+          <title>{i18n.gettext('Review Guidelines')}</title>
+        </Helmet>
+
+        <section id="review-guide">
+          <p>
+            {i18n.gettext(`Add-on reviews are a way for you to share your opinions
+              about the add-ons you’ve installed and used. Our review moderation
+              team reserves the right to refuse or remove any review that does not
+              comply with these guidelines.`)}
+          </p>
+          <h2>{i18n.gettext('Some tips for writing a great review')}</h2>
+          <h3>{i18n.gettext('Do:')}</h3>
+          <ul>
+            <li>{i18n.gettext('Write like you are telling a friend about your experience with the add-on.')}</li>
+            <li>{i18n.gettext('Keep reviews concise and easy to understand.')}</li>
+            <li>{i18n.gettext('Give specific and helpful details. For example:')}
+              <ul>
+                <li>{i18n.gettext('Did the add-on work as you expected it to?')}</li>
+                <li>{i18n.gettext('What features did you like or dislike?')}</li>
+                <li>{i18n.gettext('Was it useful?')}</li>
+                <li>{i18n.gettext('Was it easy to use?')}</li>
+                <li>{i18n.gettext('Will you continue to use this add-on?')}</li>
+              </ul>
+            </li>
+            <li>{i18n.gettext('Take a moment to read your review before submitting it to minimize typos.')}</li>
+          </ul>
+          <h3>{i18n.gettext(`Don't:`)}</h3>
+          <ul>
+            <li>{i18n.gettext('Submit one-word reviews such as "Great!", "wonderful," or "bad." ')}</li>
+            <li>{i18n.gettext(`Post technical issues, support requests, or feature suggestions. Use
+              the available support options for each add-on, if available. You can find them in
+              the side column next to the About this Add-on section.`)}</li>
```

Comment:
It's not "More information" and I'd quote "About this Add-on". So `the side column next to the "More information" section`